test/unit/controllers/subscription/WhatIsYourEoriControllerSpec.scala

```diff
@@ -94,6 +103,18 @@ class WhatIsYourEoriControllerSpec extends ControllerSpec with AuthActionMock wi
 
   "What Is Your Eori Controller" should {
 
+    "redirect to Eori number invalid page for failed CheckEoriNumber" in {
```

Comment:
Youre checking the failure path which is good but id like to see more test coverage. Happy path and what happens if you dont get a response at all?

Reply:
The other existing tests already cover the happy path (valid eori). As mentioned above the connector always returns a valid response.

Reply:
I can add a test for the exception thrown in the failed pattern match.  All depends on how we chose to handle upstream errors.  Pls let me know what you have in mind.

Reply:
Added additional test.